### ISSUE TITLE
How to configure .jshintrc per project

### ISSUE THREAD

**Issue by iegik** [georapbox/brackets-JSHint-Configurator#6]
This plugin reads only `.jshintrc` file in curent directory or parent directory, bot do not see project root directory.

**Comment by leocaseiro** [comment]
:+1: 

I'd like to choose my **.jshintrc** which is not in the root, but in a directory called /dev

**Comment by georapbox** [comment]
The plugin right now works as this: It searches the open project from the current folder to the root and if a `.jshintrc` file is found it uses this one and stops looking. If not found it will continue looking until it reaches the root of the project. This was made by design because this is how the JSHint Linter plugins work. If there are two `.jshintrc` files in a project (one in the root and one in a subfolder) and I have a file open the JSHint plugins will recognize and lint the file using the `.jshintrc` that is closer to this file as we climb the file tree towards the root.

I hope this helps?

**Comment by leocaseiro** [comment]
Thanks for letting us know.

That would be nice to set a reference to a single file optionally, like Webstorm does.

In this case, I could set my file to the .jshintrc that I want. It seems working fine for the files that are at the same directory, I'll make more tests.

Thanks again

**Comment by georapbox** [comment]
You are right about Webstorm but the last one has an integrated plugin for JSHint linting. Brackets has a couple of community plugins available but the last time I tested the two most popular, they do not allow selecting which configuration to use.

**Comment by leocaseiro** [comment]
I see, thanks for your feedback
